Runtime/Plugins/Raycast/RaycastManager.cs

```diff
@@ -0,0 +1,82 @@
+using System.Collections.Generic;
+using Unity.UIWidgets.ui;
+using UnityEngine;
+using Rect = Unity.UIWidgets.ui.Rect;
+
+namespace Unity.UIWidgets.plugins.raycast {
+    public class RaycastableRect {
+        public bool isDirty;
+        public Rect rect;
+
+        public RaycastableRect(bool isDirty) {
+            this.isDirty = isDirty;
+        }
+    }
+
+    public class RaycastManager {
+        static RaycastManager _instance;
+
+        public static RaycastManager instance {
+            get {
+                if (_instance == null) {
+                    _instance = new RaycastManager();
+                }
+
+                return _instance;
+            }
+        }
+
+        public Dictionary<int, Dictionary<int, RaycastableRect>> hashCodeList =
```

Comment:
readonly
hashCodeList is a Dictionary ? maybe rename it something like raycastHandlerMap ?